modules/gdscript/gdscript_compiler.cpp

```diff
@@ -199,8 +199,9 @@ GDScriptDataType GDScriptCompiler::_gdtype_from_datatype(const GDScriptParser::D
 		}
 	}
 
+	result.container_element_types.resize(p_datatype.container_element_types.size());
 	for (int i = 0; i < p_datatype.container_element_types.size(); i++) {
-		result.set_container_element_type(i, _gdtype_from_datatype(p_datatype.get_container_element_type_or_variant(i), p_owner, false));
+		result.container_element_types.write[i] = _gdtype_from_datatype(p_datatype.get_container_element_type_or_variant(i), p_owner, false);
```

Comment:
What this loop does with `GDScriptDataType::set_container_element_type` is virtually the same as a push back loop, so the change was done directly here.

One thing to note is that this is the only place where this function is used, so I don't know if it should be removed.

Reply:
Removing unused things is almost always good!